CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 16.0.2
```

Comment:
Since [16.0.1 has not yet released](https://github.com/uPortal-Project/uportal-app-framework/blob/master/pom.xml#L33), the changes in this changeset should continue to accumulate with the others under 16.0.1 rather than occasion 16.0.2.

Reply:
Got it1